### PR TITLE
adding a migration test

### PR DESCRIPTION
**Pull request by RonTuretzky**
addressing #7 
testing some output generated today

### CODE REVIEW

File: test/Bread.t.sol [hunk 1]
```diff
@@ -238,27 +238,279 @@ contract BreadTest is Test {
...
+            1,
+            9999000000000000000,
+            9800999999999999999,
+            1000000000000000000,
+            2238541110297335924,
+            500000000000000000,
+            23070145829119381495401
+        ];
+        breadToken.batchMint{value:polygontotalSupply}(migration_amounts, migration_addresses);
+        assertEq(breadToken.totalSupply(), polygontotalSupply + 1 ether);
```
Comment (bagelface) [inline test/Bread.t.sol:512]: would be nice to throw in some balance checks for the individual addresses here

**Review by bagelface — APPROVED** [review]
lgtm

File: test/Bread.t.sol (hunk 1 above)
Comment (RonTuretzky) [reply test/Bread.t.sol:512]: with fuzzing! opened #19

**Comment by RonTuretzky** [comment]
@bagelface added fuzzy testing for batch mint

**Pull request merged**